app/models/wiki_content/revision.rb

```diff
@@ -43,6 +43,9 @@ class Revision < ApplicationRecord
 
   include ArticleHelper
 
+  WIKITEXT_REF_TAGS = 'feature.wikitext.revision.ref_tags'
+  WIKIDATA_REFERENCES = 'feature.len(<datasource.wikidatawiki.revision.references>)'
```

Comment:
As a matter of style, since these are only used within a single method, I would usually move these constant assignments right above the method. It doesn't matter much either way, but that will be a little more consistent with the rest of the codebase.